src/fastoad/io/xml/exceptions.py

```diff
@@ -1,4 +1,4 @@
-""" Exceptions for io.xml module """
+"""Exceptions for io.xml module"""
```

Comment:
Small comment here, but there is something I find odd that ruff didn't pick up. Here, this comment doesn't end with a dot while in the __init__ of the gui folder, it does and yet both have passed. Does that mean that there is no convention for that ?

Reply:
AFAIK, there is indeed no convention about that.
I won't pretend I am consistent on this topic, but I try to end actual sentences by full stop. For "non-sentences comments", I don't feel it is mandatory.